conf/kafka_cli_jaas.conf

```diff
@@ -13,10 +13,7 @@
 //
 
 KafkaClient {
-    com.sun.security.auth.module.Krb5LoginModule required
```

Comment:
this feels odd to me. I get why the change but there should be jaas configurations for both kerberos and plaintext ( or others ). This is changing the default which probably is not the right way to go.

This comment applies to the other jaas configs as well